lib/src/widgets/links_card_widget/links_card_button.dart

```diff
@@ -8,8 +8,12 @@ class LinksCardButton extends StatefulWidget {
   /// onTap for [HoverWrapper]
   final VoidCallback onTap;
 
+  /// Sets huge modification
+  final bool huge;
```

Comment:
Same here. The sizes should probably external parameters -- makes it much easier to modify the widget look that way.